docs/maintainers/generic-how-to-implement-a-concept-exercise.md

```diff
@@ -77,13 +77,17 @@ Having added the files that are not specific to the track, now is the time to cr
 
 What these files look like depends on your track. Note that some tracks might require more files in addition to the three files just mentioned.
 
-## Step 8: add analyzer (optional)
+## Step 8: update the general concept document
+
+Add the exercise to the [concept's shared document's][reference] `## Implementations` section.
```

Comment:
I like this addition. Maybe it would be useful to link one of the existing documents (I think the `strings` document has a section with contents)?

Reply:
I don't mind adding it, but none of the other steps have examples.

Reply:
That is correct, but in this case I think it makes sense as I'm slightly worried new maintainers would not immediately understand what it is referring to.